### ISSUE TITLE
usando os dados serial de forma direto sem o modem do fabricante 

### ISSUE THREAD

**Issue by manotroll** [krywenko/WVC-inverters#3]
It is possible to use without the manufacturer's modem it is 433mhz
is it possible to use on windows or just linux?
I need an orange pi I can't use anything else like esp32 or stm32

**Comment by krywenko** [comment]
yes  see this version https://github.com/krywenko/WVC-Inverter-R2-R3-HC-12--and--Modem you can use  linux , BSD openwrt.  or what have you you could probably do it  esp or esp 32 or stm32 pretty much any OS but you have to modify the code to work
